htdocs/compta/facture/card.php

```diff
@@ -1287,7 +1287,15 @@
 						$totaldeposits = $facture_source->getSumDepositsUsed();
 						$remain_to_pay = abs($facture_source->total_ttc - $totalpaid - $totalcreditnotes - $totaldeposits);
 
-						$object->addline($langs->trans('invoiceAvoirLineWithPaymentRestAmount'), $remain_to_pay, 1, 0, 0, 0, 0, 0, '', '', 'TTC');
+						if ((empty($soc) || !is_object($soc) || get_class($soc) != 'Societe') && $socid >0) {
+							$soc = new Societe($db);
+							$soc->fetch($socid);
```

Comment:
Don't you already have the thirdparty into $object->thirdparty ?

Reply:
sorry no, but I'll use a more elegant method ..